src/python/turicreate/toolkits/style_transfer/style_transfer.py

```diff
@@ -163,7 +161,8 @@ def create(style_dataset, content_dataset, style_feature=None,
         'aug_inter_method': 2,
         'checkpoint': False,
         'checkpoint_prefix': 'style_transfer',
-        'checkpoint_increment': 1000
+        'checkpoint_increment': 1000,
+        'use_tensorflow': True # TODO: Change to False
```

Comment:
Let's change to false before we integrate. By default, TF should not be imported at all